tools/lib/h5tools.c

```diff
@@ -671,12 +671,8 @@ h5tools_set_fapl_vol(hid_t fapl_id, h5tools_vol_info_t *vol_info)
                 /* Check for VOL connectors that ship with the library, then try
                  * registering by name if that fails.
                  */
-                if (!strcmp(vol_info->u.name, H5VL_NATIVE_NAME)) {
```

Comment:
This looks like it would cause problems when trying to use h5tools with the native VOL